float32 x
float32 y
float32 time_res
---
float32[] v
float32[] omega